builders/pallets-precompiles/precompiles/proxy.md

```diff
@@ -0,0 +1,124 @@
+
+## Interact with the Solidity Interface {: #interact-with-the-solidity-interface }
+
+### Checking Prerequisites {: #checking-prerequisites } 
+
+The below example is demonstrated on Moonbase Alpha, however, similar steps can be taken for Moonbeam and Moonriver. You should:  
+
+ - Have MetaMask installed and [connected to Moonbase Alpha](/tokens/connect/metamask/){target=_blank}
+ - Have an account with some DEV tokens.
+ - Have a second account that you control to use as a proxy account (funding optional).
+ --8<-- 'text/faucet/faucet-list-item.md'
+
+### Remix Set Up {: #remix-set-up } 
+
+1. Get a copy of [`Proxy.sol`](https://github.com/PureStake/moonbeam/blob/master/precompiles/proxy/Proxy.sol){target=_blank}
+2. Copy and paste the file contents into a [Remix file](https://remix.ethereum.org/){target=_blank} named `Proxy.sol`
+
+![Copying and Pasting the Proxy Interface into Remix](/images/builders/pallets-precompiles/precompiles/proxy/proxy-1.png)
+
+### Compile the Contract {: #compile-the-contract } 
+
+1. Click on the **Compile** tab, second from top
+2. Then to compile the interface, click on **Compile Proxy.sol**
+
+![Compiling Proxy.sol](/images/builders/pallets-precompiles/precompiles/proxy/proxy-2.png)
+
+### Access the Contract {: #access-the-contract } 
+
+1. Click on the **Deploy and Run** tab, directly below the **Compile** tab in Remix. Note: you are not deploying a contract here, instead you are accessing a precompiled contract that is already deployed
+2. Make sure **Injected Web3** is selected in the **ENVIRONMENT** drop down
+3. Ensure **Proxy.sol** is selected in the **CONTRACT** dropdown. Since this is a precompiled contract there is no need to deploy, instead you are going to provide the address of the precompile in the **At Address** Field
+4. Provide the address of the proxy precompile for Moonbase Alpha: `{{networks.moonbase.precompiles.proxy}}` and click **At Address**
+5. The proxy precompile will appear in the list of **Deployed Contracts**
+
+![Provide the address](/images/builders/pallets-precompiles/precompiles/proxy/proxy-3.png)
+
+### Add a Proxy {: #add-proxy } 
+
+You can add a proxy for your account via the proxy precompile if your account doesn't already have a proxy. In this example, you will add a Balances proxy to an account by taking the following steps:
+
+1. Expand the proxy precompile contract to see the available functions
+2. Find the **addProxy** function and press the button to expand the section
+3. Insert your second account's address as the *delegate*, `5` as *proxyType*, `0` and as *delay*
+4. Press **transact** and confirm the transaction in MetaMask
+
+!!! note
+     When constructing the transaction in Remix, the *proxyType* is represented as a `uint8`, instead of the expected enum `ProxyType`. In Solidity, enums are compiled as `uint8`, so when you pass in `5` for *proxyType*, you indicate the sixth element in the `ProxyType` enum, which is the Balances proxy.
```

Comment:
```suggestion
     When constructing the transaction in Remix, the **proxyType** is represented as a `uint8`, instead of the expected enum `ProxyType`. In Solidity, enums are compiled as `uint8`, so when you pass in `5` for **proxyType**, you indicate the sixth element in the `ProxyType` enum, which is the balances proxy.
```